sceptre/plan/plan.py

```diff
@@ -376,3 +376,13 @@ def diff(self, *args) -> Dict[Stack, StackDiff]:
         """
         self.resolve(command=self.diff.__name__)
         return self._execute(*args)
+
+    def drift_show(self, *args):
+        """
+        Show stack drift for a running stack.
+
+        :returns: A list of detected drift against running stacks.
+        :rtype: List[str]
```

Comment:
This method doesn't actually return a list of strings.

Reply:
Oops, fixed.